rcsb/utils/chem/ChemCompSearchIndexProvider.py

```diff
@@ -87,7 +88,8 @@ def __buildChemCompSearchIndex(self, procName, ccIdList, descrD, limitPerception
                     oemf.setQuiet()
                 tId = oemf.setChemCompDef(dataContainer)
                 if tId != ccId:
-                    logger.error("%s %s chemical component definition import error", procName, ccId)
+                    if ccId != "UNL":  # Expecting this to fail for UNL, so only log error if it's not
+                        logger.error("%s %s chemical component definition import error", procName, ccId)
                     fL.append(ccId)
```

Comment:
@piehld is this step required for "UNL" or should it be skipped as well?

Reply:
Sorry it's hard to tell which line you commented on—which line are you referring to?

Reply:
I am referring to `fL.append(ccId)`.

Reply:
OK thanks for clarifying. For that I'm not sure. I decided to keep it here in case that does still need to happen, and in previous context that step was still being run as I believe the code block continues even after a `logger.error` occurs. It'll require some more testing to see if that's really needed or not.

Reply:
Perhaps ok to leave it as is for now.